src/backend/joanie/tests/core/test_api_admin_course_access.py

```diff
@@ -0,0 +1,329 @@
+"""
+Test suite for CourseAccess Admin API.
+"""
+import uuid
+
+from django.test import TestCase
+
+from joanie.core import enums, factories
+
+
+class CourseAccessAdminApiTest(TestCase):
+    """
+    Test suite for CourseAccess Admin API.
+    """
+
+    def test_admin_api_course_accesses_request_anonymous(self):
+        """
+        Anonymous users should not be able to request course accesses endpoint.
+        """
+        course = factories.CourseFactory()
+        response = self.client.get(f"/api/v1.0/admin/courses/{course.id}/accesses/")
+
+        self.assertEqual(response.status_code, 403)
+        content = response.json()
+        self.assertEqual(
+            content["detail"], "Authentication credentials were not provided."
+        )
+
+    def test_admin_api_course_accesses_request_authenticated(self):
+        """
+        Authenticated users should not be able to request course accesses endpoint.
+        """
+        admin = factories.UserFactory(is_staff=False, is_superuser=False)
```

Comment:
If the user is not an admin, it would be more appropriate to call it just `user`

Reply:
Indeed! Copy/pasta failed